### PR TITLE
ISSUE_TRIAGE.md: update

### PR DESCRIPTION
**Pull request by tonypowa**
- added current bug issue template bullet points to # Bug reports subsection
- updated enhancement requests' subsection by adding info regarding GH discussions
- added link to WCAG criteria in Accessibility issues subsection
- linked Support requests' subsection  to section  # 3 Categorizing an issue/#support-requests subsection

### CODE REVIEW

**Comment by natellium** [comment]
Looking good, just a couple of comments :)

**Review by natellium — CHANGES_REQUESTED** [review]
It's looking good, left a couple of suggestions

File: ISSUE_TRIAGE.md [hunk 1]
```diff
@@ -87,28 +87,27 @@ Given a certain [issue template](https://github.com/grafana/grafana/issues/new/c
...
-Should explain what enhancement or feature that the author wants to be added and why that is needed.
+Enhancement requests (aka. feature requests) should first be discussed in [Github discussions](https://github.com/grafana/grafana/discussions). When submitting an idea for discussion we welcome users to create a new discussion and use these prompts:
+ 
+- What would you like to be added
+- Why is this needed (describe your use case and goals)**:
 
 #### Accessibility issues
 
-This is a mix between a bug report and enhancement request but focused on accessibility issues to help make Grafana improve keyboard navigation, screen-reader support and being accessible to everyone. The report should include relevant WCAG criteria, if applicable.
+This is a mix between a bug report and enhancement request but focused on accessibility issues to help make Grafana improve keyboard navigation, screen-reader support and being accessible to everyone. The report should include relevant [WCAG criteria](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0), if applicable.
```
[the remark this answers is not in the document]
Comment (tonypowa) [reply ISSUE_TRIAGE.md:106]: Thanks for the suggestions. Let me know if the new proposal works :)

**Review by mellieA — COMMENTED** [review]
What do you think about adding a table for the bot actions in automation? 

It looks like the current version has a link to this page: 
Screenshot 2022-12-16 at 10 45 22 AM

**Review by mellieA — COMMENTED** [review]
**Investigation of issues** section: 
The new label specific to triage in place of `needs investigation` is [`triage/needs-confirmation`](https://github.com/grafana/grafana/issues?q=is%3Aissue+is%3Aopen+label%3Atriage%2Fneeds-confirmation)

**Comment by tonypowa** [comment]
@natellium  I think I am done updating the whole document. Awaiting your review 😸

**Review by natellium — CHANGES_REQUESTED** [review]
👍 Good updates, left a couple of comments. The flow chart doesn't make sense tho, I'd suggest spending some time understanding what the new flow is and mapping it from scratch.

File: ISSUE_TRIAGE.md [hunk 2]
```diff
@@ -308,19 +330,28 @@ Even if you don't have the time or knowledge to investigate an issue we highly r
 
 We have some automation that triggers on comments or labels being added to issues. Many of these automated behaviors are defined in [commands.json](https://github.com/grafana/grafana/blob/main/.github/commands.json). Or in other [GitHub Actions](https://github.com/grafana/grafana/tree/main/.github/workflows)
 
-- Add /duplicate `#<issue number>` to have Grafana label & close issue with an appropriate message.
-- Add `bot/question` and the bot will close it with an appropriate message.
+Comment commands:
```
[the remark this answers is not in the document]
Comment (tonypowa) [reply ISSUE_TRIAGE.md:333]: Yeah, let's keep it less busy. Agreed !

**Comment by tonypowa** [comment]
@natellium resolved pending comments. Hope it is good to go! :)

**Review by natellium — APPROVED** [review]
LGTM, good work! :)

**Comment by jdbaldry** [comment]
Thanks for the PR. I'm not sure that I'm in the best position to review this document as I don't actively triage issues in this repository. If you'd still like me to take a look, ping me and I'll make sure to do so :)

**Pull request merged**